inc/File/Resolver.php

```diff
@@ -80,6 +80,7 @@ protected function resolvePrefix($id)
      */
     protected function resolveRelatives($id)
     {
+        $id = trim($id, '.'); // trailing dots are invalid
```

Comment:
does this only remove the dots on the end of the pageid? normally `trim()` acts on the begin and the end. (as the unit test passes, I guess it works, but I did not expect that from reading this code. If the intend is only removing trailing dots, `rtrim()` is more showing what is meant?

I guess it works because the prefixed dots should be resolved already at this point.

Reply:
Absolutely! This was a typo that only accidentally still worked.